packages/terra-card/src/Card.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import './Card.scss';
+
+const propTypes = {
+  /**
+   * Provides 10px on top and bottom
+   */
+  padVertical: PropTypes.bool,
+  /**
+   * Provides 10px on left and right
+   */
+  padHorizontal: PropTypes.bool,
+  /**
+   * Child Nodes
+   */
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  padVertical: true,
+  padHorizontal: true,
```

Comment:
the defaults should be no padding per this
https://github.com/cerner/terra-core/issues/313#issuecomment-322214889